modules/flowable-engine/src/test/java/org/flowable/engine/test/bpmn/event/escalation/BoundaryEscalationEventTest.java

```diff
@@ -12,12 +12,16 @@
  */
 package org.flowable.engine.test.bpmn.event.escalation;
 
+import static java.lang.String.format;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
Where as there are about 6 uses of `assertThrows` in the code base, the primary form used for this type of test is `org.assertj.core.api.Assertions.assertThatThrownBy;` with 1000+ uses.  Please consider changing to the more common format.  Thanks.

Reply:
Feedback incorporated. Thank you!